src/Ui.Playground/Internal/PlaygroundPageModel.cs

```diff
@@ -20,19 +19,16 @@ public string Render()
         {
             if (_playgroundCSHtml == null)
             {
-                using (var manifestResourceStream = typeof(PlaygroundPageModel).Assembly.GetManifestResourceStream("GraphQL.Server.Ui.Playground.Internal.playground.cshtml"))
-                {
-                    using (var streamReader = new StreamReader(manifestResourceStream))
-                    {
-                        var builder = new StringBuilder(streamReader.ReadToEnd());
-
-                        builder.Replace("@Model.GraphQLEndPoint", _options.GraphQLEndPoint);
-                        builder.Replace("@Model.GraphQLConfig", JsonConvert.SerializeObject(_options.GraphQLConfig));
-                        builder.Replace("@Model.PlaygroundSettings", JsonConvert.SerializeObject(_options.PlaygroundSettings));
-
-                        _playgroundCSHtml = builder.ToString();
-                    }
-                }
+                using var manifestResourceStream = typeof(PlaygroundPageModel).Assembly.GetManifestResourceStream("GraphQL.Server.Ui.Playground.Internal.playground.cshtml");
```

Comment:
Just a note - I am not a fan of C#8 using declarations but I have nothing against using it in small methods like this one. By the way, if you used them in Playground and Altair projects, then let's do it in the other two UI projects.

Reply:
Yea I'm not a fan in all cases either, I like to keep them short-lived, but this one made sense.